plone/transformchain/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
Well, i usually like to have it in there, even if superfluous now, but what if someone starts adding code there?